controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -310,15 +310,32 @@ func (b *builder) addTransformStages(stage *config.PipelineBuilderStage) (*corev
 	lastStage := *stage
 	indexFields := constants.LokiIndexFields
 
+	var clusterName string
+	if b.desired.Processor.ClusterName != "" {
+		clusterName = b.desired.Processor.ClusterName
+	} else {
+		//take clustername from openshift
+		clusterName = string(constants.DefaultClusterID)
+	}
+	transformFilterRules := []api.TransformFilterRule{
+		{
+			Input: "clusterName",
```

Comment:
We should use the same pattern as the other fields, ie:

```suggestion
			Input: "K8S_ClusterName",
```

Reply:
done